modules/Microsoft.Batch/batchAccounts/applications/deploy.bicep

```
@@ -0,0 +1,54 @@
+@description('Required. Name of the batch Account.')
+param batchName string
```

I'd suggest to change to
```suggestion
param batchAccountName string
```
For consistency. Here and for related referencies.